src/pages/profiles/forms/ProfileDetailsForm.tsx

```diff
@@ -19,10 +19,13 @@ export const profileDetailPayload = (values: CreateProfileFormValues) => {
 
 interface Props {
   formik: FormikProps<CreateProfileFormValues>;
+  isEdit?: boolean;
 }
 
-const ProfileDetailsForm: FC<Props> = ({ formik }) => {
+const ProfileDetailsForm: FC<Props> = ({ formik, isEdit = true }) => {
```

Comment:
This should be `false` by default, no? Given that you're not passing it to the `CreateProfileForm`.

Reply:
true! I'll better make it non-optional to avoid confusion on it.